### ISSUE TITLE
JAWS not handling Complex Table id/header association properly

### ISSUE THREAD

**Issue by goodwitch** [FreedomScientific/VFO-standards-support#678]
### Summary
JAWS is not respecting the id/header association properly in complex data tables. 

Example:
1. Go to http://www.glendathegood.com/a11y/complextable.html
2. Use JAWS/Chrome and/or JAWS/Firefox 
3. Listen to the "Supplier contacts" table and go to the data cell with value "Marie Beauchamp" and note that JAWS/Chrome and JAWS/Firefox does not respect the header/id association which is "Example 2 Co".

### Expected result

JAWS should have only associated the table header "Example 2 Co" with the data cell "Marie Beauchamp"

### Actual result

> Jaws says: Column 3. Row 2. Example 2 Co. Example 4 Inc. Contact. Marie Beauchamp.

**Comment by JAWS-test** [comment]
To put it more precisely: JAWS ignores the header attribute completely in every table (and not only in complex tables)

**Comment by donraikes** [comment]
This has been fixed in JAWS 2023.
